raspberry/checkout/checkout_main.py

```diff
@@ -31,6 +36,8 @@ def main(self):
         self.communications.assign_response_action(self.server_response_received)
         self.communications.on_start()
 
+        self.interactions.start_rfid_listener(self.process_rfid_card)
```

Comment:
im worried about the start listener function. I have a strong suspicion that it will not be able to recieve response messages if the program is one small loop. Same goes for pressing the buttons. we will need to check if pressing the button even registers while the loop is running

Reply:
everything works fine, nothing to resolve.